middle/isq.ttl

```diff
@@ -12,36 +12,36 @@
                                     owl:versionIRI <http://emmo.info/emmo/1.0.0-beta/middle/isq> ;
                                     owl:imports <http://emmo.info/emmo/1.0.0-beta/middle/materials> ,
                                                 <http://emmo.info/emmo/1.0.0-beta/middle/metrology> ;
-                                    dcterms:abstract """European Materials & Modelling Ontology (EMMO)
-
-EMMO is a multidisciplinary effort to develop a standard representational framework (the ontology) based on current materials modelling knowledge, including physical sciences, analytical philosophy and information and communication technologies.
-
-It provides the connection between the physical world, materials characterisation world and materials modelling world.
-
-EMMO is released under a Creative Commons license Attribution 4.0 International (CC BY 4.0)."""@en ;
-                                    dcterms:contributor "Access, DE" ,
-                                                        "Fraunhofer IWM, DE" ,
-                                                        "Goldbeck Consulting Ltd (UK)" ,
+                                    dcterms:title "European Materials & Modelling Ontology"@en ;
+                                    dcterms:creator "Adham Hashibon" ;
+                                    dcterms:contributor "Goldbeck Consulting Ltd (UK)" ;
+                                    dcterms:publisher "EMMC ASBL" ;
+                                    dcterms:creator "Georg Schmitz" ;
+                                    dcterms:contributor "Fraunhofer IWM, DE" ,
                                                         "SINTEF, NO" ,
                                                         "University of Bologna, IT" ;
-                                    dcterms:creator "Adham Hashibon" ,
-                                                    "Emanuele Ghedini" ,
-                                                    "Georg Schmitz" ,
-                                                    "Gerhard Goldbeck" ,
-                                                    "Jesper Friis" ;
-                                    dcterms:license "https://creativecommons.org/licenses/by/4.0/legalcode" ;
-                                    dcterms:publisher "EMMC ASBL" ;
-                                    dcterms:title "European Materials & Modelling Ontology"@en ;
+                                    dcterms:creator "Emanuele Ghedini" ;
```

Comment:
This seems to change the "creator" and "contributor" list, is this intended?

Reply:
This is an unintended and unwanted result of working with the ontology in tools like Protege. The above change is just syntactic due to Protege either serialising in a somewhat arbitrary order or, as in this case, different versions of Protege serialises with slightly different syntax.

Not sure how to avoid it.

Reply:
Right. The only thing is to use the same version as Protégé always perhaps?

Otherwise, if there are no actual changes, and all intended creators and contributors are still correctly listed in Protégé when importing the ontology, then it shouldn't matter. I just thought it was weird and wanted to make sure it didn't suddenly remove someone. Indeed, it seems to me that yourself and Gerhard has been removed as creators?

Reply:
I think we are added as contributors further down the diff. But thank you for checking. It is actually owlapi that we need to use the same version of, which is little more tricky to ensure.

Reply:
It's worth noting this down for developers in a general place to be aware of these git diff challenges and file changes when working with ontologies in Protégé.